src/components/appNavigation/PushButton.js

```diff
@@ -23,13 +23,13 @@ type PushButtonProps = {
  * @param {ButtonProps} props
  */
 export const PushButton = ({ routeName, screenProps, canContinue, params, ...props }: PushButtonProps) => {
-  const OnPress = useOnPress(async () => screenProps && (await canContinue()) && screenProps.push(routeName, params), [
+  const onPress = useOnPress(async () => screenProps && (await canContinue()) && screenProps.push(routeName, params), [
```

Comment:
one-liner looks weird here. pls use multi-line syntax, without && (e.g. just if )

this comment relates to the **all places** where on-liner with complex conditions is used
use one-liner if:
- you need to wrap function referenced via props or defined above
- you need to wrap simple call of function (e.g. navigation.navigate)
- you have one simple if condition - use && in. this case
- you have one simple if-else condition - use ? : in this case

Reply:
done